content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,408 @@
+to automatically inject the `spire` template, see [custom templates](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#custom-templates-experimental),
+in the configuration above into new workloads pods, or add the pod annotation to your deployment spec:
+
+{{< text yaml >}}
+podAnnotations:
+  inject.istio.io/templates: "sidecar,spire"
+{{< /text >}}
+
+This will share the spiffe-csi-driver with the Ingress Gateway and the sidecars that are going to be automatically injected on workload pods,
+granting them access to the SPIRE Agent's UNIX Domain Socket.
+
+Check istio-ingressgateway pod state:
+
+{{< text bash >}}
+$ kubectl get pods -n istio-system
+NAME                                    READY   STATUS    RESTARTS   AGE
+istio-ingressgateway-5b45864fd4-lgrxs   0/1     Running   0          17s
+istiod-989f54d9c-sg7sn                  1/1     Running   0          23s
+{{< /text >}}
+
+Data plane containers will only reach the `Ready` if a corresponding registration entry is created for them on the SPIRE Server. Then,
+Envoy will be able to fetch cryptographic identities from SPIRE.
+See [Register workloads](#register-workloads) to register entries for services in your mesh.
+
+## Register workloads
+
+This section describes the options available for registering workloads in SPIRE Server.
+
+### Option 1: Automatic registration using the SPIRE workload registrar
+
+By deploying [SPIRE Kubernetes Workload Registrar](https://github.com/spiffe/spire/tree/main/support/k8s/k8s-workload-registrar)
+along with SPIRE Server, new entries are automatically registered for each new pod that is created.
+
+See [Verifying that identities were created for workloads](#verifying-that-identities-were-created-for-workloads) 
+to check issued identities.
+
+### Option 2: Manual Registration
+
+SPIRE Server is able to verify a group of selector values to improve workload attestation security robustness.
+
+1. To generate an entry for an Ingress Gateway with a set of selectors such as the
+   pod name and pod UID:
+
+{{< text bash >}}
+$ INGRESS_POD=$(kubectl get pod -l istio=ingressgateway -n istio-system -o jsonpath="{.items[0].metadata.name}")
+$ INGRESS_POD_UID=$(kubectl get pods -n istio-system $INGRESS_POD -o jsonpath='{.metadata.uid}')
+{{< /text >}}
+
+2. Get the spire-server pod:
+
+{{< text bash >}}
+$ SPIRE_SERVER_POD=$(kubectl get pod -l app=spire-server -n spire -o jsonpath="{.items[0].metadata.name}")
+{{< /text >}}
+
+3. Attest the SPIRE Agent running on the node:
+{{< text bash >}}
+$ kubectl exec -n spire $SPIRE_SERVER_POD -- \
+/opt/spire/bin/spire-server entry create \
+-spiffeID spiffe://example.org/ns/spire/sa/spire-agent \
+-selector k8s_psat:cluster:demo-cluster \
+-selector k8s_psat:agent_ns:spire \
+-selector k8s_psat:agent_sa:spire-agent \
+-node -socketPath /run/spire/sockets/server.sock

```

Comment:
I'm being picky, but what do you think about indenting from the 2nd line in all commands for easier reading?